ui/partials/main/jobs.html

```diff
@@ -16,8 +16,10 @@
     <span class="result-set-left">
       <span class="result-set-title-left">
         <span>
-          <a href="{{::revisionResultsetFilterUrl}}"
+          <a ng-href="{{::revisionResultsetFilterUrl}}"
+             ng-click="changeResultSet(repoName, resultset.revision)"
              title="View only this resultset"
+             prevent-default-on-left-click
```

Comment:
Sorry about my ambiguity on that.  I was referring to ``revisionResultsetFilterUrl`` which gives the url  to the href for filtering down to a single revision.  We'd need to add the filter params to that URL to get this to work like a regular href.  

I'm not sure how expensive it would be to actually update that var in every push every time a filter changed.  Maybe not that bad.  It would need to be changed to NOT use the ``::`` one-way binding prefix in that case.

Actually, probably making is something like this would work:

    <a ng-href="{{::revisionResultsetFilterUrl}}{{filterParams}}"

Just need to set ``filterParams`` in a place that's accessible to all the pushes, which shouldn't be too hard.